### ISSUE TITLE
Annoying empty banner in the new layout

### ISSUE THREAD

**Issue by RichardLeblanc** [mrpond/BlockTheSpot#147]
![post](https://user-images.githubusercontent.com/47771443/114280967-05be3980-9a3c-11eb-92c7-68ad5b93b4e8.png)

As I said there is a very annoying empty banner (the one where I put the red arrow obviously) displayed in artists page, saved album page and all the rest except in the home.

Home screenshot without the banner:

![post 2](https://user-images.githubusercontent.com/47771443/114281168-1fac4c00-9a3d-11eb-8854-8295d36e4b8e.png)

 - OS: Windows 10
  1. Latest Spotify version 1.1.56.595.g2d2da0de
  2. Latest blockthespot from github

**Comment by rednek46** [comment]
You are using an Alpha version of Spotify which is not currently supported by BlockTheSpot.

Use BlockTheSpot.bat and enter Y when prompt appears to swap to OLD UI. and it will work well.

**Comment by orestbida** [comment]
@rednek46 this update was automatically executed on my client. Do you plan on adding support for the new layout?

**Comment by rednek46** [comment]
It will be supported once Spotify finishes with the alpha testing.

**Comment by RichardLeblanc** [comment]
> 
> 
> @rednek46 this update was automatically executed on my client. Do you plan on adding support for the new layout?

Yes it is automatic, I didn't update my client (I disabled auto updates with the strings in file hosts) I just run Spotify and then boom it was with the new UI, plus whatever older build I try to install over, it will come with the "-a" even if it's a stable version. However I have no ads at all and all features are working well, except that annoying banner.
I found a "solution" installing a old build "1.1.48.625.g1c87c7f7" that automatically became "1.1.48.625.g1c87c7f7-a" as I said before but the strange thing is that with that build the UI is a mix with the old and the new UI and the annoying banner is gone.
To me now is just perfect and I think I will keep this build. I have tried to remove the upgrade button too but with no success.

![post](https://user-images.githubusercontent.com/47771443/114305338-0f957a80-9ad8-11eb-87f0-b638e2cb5fe2.png)
